kuma/core/helpers.py

```diff
@@ -150,8 +151,11 @@ def entity_decode(str):
 
 
 @register.function
-def inlinei18n(locale):
-    return StaticI18nJob().get(locale)
+def statici18n(locale):
+    if not locale:
+        locale = settings.LANGUAGE_CODE
+    filename = get_filename(locale, settings.STATICI18N_DOMAIN)
+    return static(os.path.join(settings.STATICI18N_OUTPUT_DIR, filename))
```

Comment:
Docs say, "By default, the generated catalogs are stored to STATIC_ROOT/jsi18n. You can modify the output path and more options by tweaking django-statici18n settings." So, I'd say yes.

Technically we don't want to move the `.po` files, but the generated `.js` files that are output from `compilejsi18n`.

Is `build/locale` at our root directory? E.g. you want these output as `build/es/javascript.js`?